src/content/layout/js_layout.h

```diff
@@ -34,6 +34,7 @@
 
 #include <memory>
 
+#include "content/scripting/import_script.h"
```

Comment:
If you add the include, please adjust the forward declarations.

Reply:
Removed all of them as import_script.h has them.

The actual reason for this change was the destructor removal. make_unique needs a declaration of ImportScript